thunder/core/transform_common.py

```diff
@@ -111,6 +111,49 @@ def check(inp, log_str):
             check(copy_to_out, "output")
 
 
+def _replace_copy_output_with_source(extrace: Trace) -> None:
+    """This is based on the sharp edge of nvfuser's `add_ouput(output, input)` interface,
```

Comment:
@jjsjann123, could you please review the changes here? Is this still a sharp edge in nvFuser? Is the target transformation correct?

Reply:
@jjsjann123 If you would please re-review this change...